src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -55,6 +78,16 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, Messages.format(personToDelete)));
     }
 
+    private CommandResult deleteByIndex(Model model) throws CommandException {
+        List<Person> lastShownList = model.getFilteredPersonList();
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
```

Comment:
Using ```getOneBased() > lastShownList.size()``` may be more intuitive, but i see why zerobased was used for consistency on the next line